RELEASE-NOTES.txt

```diff
@@ -1,3 +1,24 @@
+1.3.3
+=====
+* Handle zip imports regression #874
+* Sentry: change release naming #868
+* Sentry: improve breadcrumbs with providing more info #864
+* Sentry: add user identifier #863
+* Show warning when user tries to pull a site that is larger than 2GB #842
+* Remove internal mu-plugins during prepareWordPress #860
```

Comment:
Not user-facing. Mostly just fixed a bug from #845

Reply:
Only Sentry related points merged to 1 line, other removed